### DISCUSSION TITLE
Discord Community

### DISCUSSION THREAD

**Discussion by redstonepro109** [discord-jda/JDA#2057] [Questions and Help] [answered]
How do i enable and disable discord community using JDA

**Comment by RedcodesDev** [comment, marked as answer]
Bots currently can't enable the community feature on a guild. An Administrator of the guild must enable it manually. You may check if the guild has the community feature enabled by using guild#getFeatures() https://ci.dv8tion.net/job/JDA5/javadoc/net/dv8tion/jda/api/entities/Guild.html#getFeatures().

**Reply by redstonepro109** [reply]
That still helps thank you 🙏
